### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by timschofield** [timschofield/webERP#185] [Announcements]
Dear all,

The website at [weberp.org](http://weberp.org/) had been deleted and so I have now created a
new one at https://weberp.org/. The code for this site is at
https://github.com/timschofield/weberp.org if anybody has any ideas
for improvements/fixes, please create pull requests. In the deletion
the forum was lost, and with it 12 years of community contributed
wisdom. I am doing my best to scrape this info back, and put it up on
the website.

The demo doesn't work at the moment. I have been working hard at many
changes to webERP (https://github.com/timschofield/webERP/wiki/V5) in
order to release a version 5.0 in the near future, and it is my
intention to create a release candidate for v5, and when I have it, I
will put it up on the site as a demo for wny interested people to
check. The current code for v5.0 can be found at
https://github.com/timschofield/webERP

Thanks
Tim

**Comment by juergenmueller77** [comment]
Looking good Tim :+1: 
If I get a chance over the weekend I will see if I can help on the website
Juergen
